packages/eslint-plugin/docs/rules/no-unnecessary-type-constraint.md

```diff
@@ -7,8 +7,8 @@ description: 'Disallow unnecessary constraints on generic types.'
 > See **https://typescript-eslint.io/rules/no-unnecessary-type-constraint** for documentation.
 
 Generic type parameters (`<T>`) in TypeScript may be "constrained" with an [`extends` keyword](https://www.typescriptlang.org/docs/handbook/generics.html#generic-constraints).
-When no `extends` is provided, type parameters default a constraint to `any`.
-It is therefore redundant to `extend` from `any`.
+When no `extends` is provided, type parameters default a constraint to `unknown`.
+It is therefore redundant to `extend` from `unknown`.
```

Comment:
[Phrasing] Ah, the rule reports both on `any` and `unknown` as of TypeScript 3.9:

https://github.com/typescript-eslint/typescript-eslint/blob/09c04fb0d5a5dee0dc266abb2e21aa9a1c489712/packages/eslint-plugin/src/rules/no-unnecessary-type-constraint.ts#L56-L61

How about ...

```suggestion
It is therefore redundant to `extend` from `any` or `unknown`.
```

Reply:
Yeah, make sense :+1: